.github/CODEOWNERS

```diff
@@ -0,0 +1 @@
+* @jenkins-infra/kubernetes
```

Comment:
```suggestion
* @jenkins-infra/kubernetes

```

Reply:
Nitpick: As per POSIX, all lines should end with an "endline" character (https://stackoverflow.com/questions/729692/why-should-text-files-end-with-a-newline -> https://pubs.opengroup.org/onlinepubs/9699919799/basedefs/V1_chap03.html#tag_03_206)